sci-libs/ccfits/ccfits-2.5.recipe

```diff
@@ -0,0 +1,114 @@
+SUMMARY="C++ Wrappers for the cfitsio library"
+DESCRIPTION="CCfits is an object oriented interface to the cfitsio library. \
+It is designed to make the capabilities of cfitsio available to programmers \
+working in C++. It is written in ANSI C++ and implemented \
+using the C++ Standard Library with namespaces, exception handling, \
+and member template functions."
+HOMEPAGE="https://heasarc.gsfc.nasa.gov/fitsio/CCfits"
+COPYRIGHT="2002-2019 Ben Dorman, Craig Gordon, Bryan Irby - HEASARC, NASA/GSFC"
+LICENSE="CCFITS"
+REVISION="1"
+SOURCE_URI="https://heasarc.gsfc.nasa.gov/fitsio/CCfits/CCfits-$portVersion.tar.gz"
+CHECKSUM_SHA256="938ecd25239e65f519b8d2b50702416edc723de5f0a5387cceea8c4004a44740"
+SOURCE_DIR="CCfits"
+PATCHES="ccfits-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 x86_64 ?arm ?ppc ?sparc"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="0.0.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	ccfits$secondaryArchSuffix = $portVersion compat >= 2.5
+	lib:libccfits$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libcfitsio$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	ccfits${secondaryArchSuffix}_devel = $portVersion compat >= 2.5
+	devel:libCCfits$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES_devel="
+	ccfits$secondaryArchSuffix == $portVersion base
+	haiku$secondaryArchSuffix
+	lib:libcfitsio$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libcfitsio$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:automake
+	cmd:awk
+	cmd:gcc$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	"
+
+TEST_REQUIRES="
+	cmd:test
+	"
+
+defineDebugInfoPackage ccfits$secondaryArchSuffix \
+	"$libDir"/libCCfits.so.$libVersion
+
+PATCH()
+{
+	mv configure.in configure.ac || true
+}
```

Comment:
Is this needed? autoconf maybe gives a warning to use configure.ac over configure.in, but haven't encountered problems with them so far ...

Reply:
Not needed, but why not saving a warning if you can do that so easily (Gentoo port does, actually)?